### ISSUE TITLE
Deadlinks to Validator source

### ISSUE THREAD

**Issue by Cabalist** [tlvince/w3c-validator-guide#17]
Both `http://validator.w3.org/validator.tar.gz` and  `http://validator.w3.org/sgml-lib.tar.gz` return 404s.

I believe the new link that should be used is:
https://github.com/w3c/markup-validator/archive/validator-1_3-release.tar.gz

This includes the latest sgml-lib files.

Section 2 is now.  Note the lack of smgl-lib line

---
### 2. Install Validator

Note: optionally on this step you can choose to download a more up-to-date DTD library from the development repository at http://dvcs.w3.org/hg/markup-validator/file/tip/htdocs/sgml-lib
1. Download the Validator and the sgml-lib DTD library:
   
   ```
   cd ~/build
   wget https://github.com/w3c/markup-validator/archive/validator-1_3-release.tar.gz
   tar -xzf validator-1_3-release.tar.gz
   ```
2. Install the Validator:
   
   ```
   sudo mkdir -p /usr/local/validator
   sudo mv markup-validator-validator-1_3-release/httpd/cgi-bin /usr/local/validator
   sudo mv markup-validator-validator-1_3-release/{htdocs,share,httpd} /usr/local/validator
   ```

**Comment by tlvince** [comment]
Would you be able to file a pull request for this (and the others)? Thanks.

**Comment by Cabalist** [comment]
Done.  Sorry for the delay.  The notice got lost in my inbox.

**Comment by tlvince** [comment]
Closed by #18.
